Src/BootCamp.Chapter/Items/Headpiece.cs

```diff
@@ -0,0 +1,16 @@
+﻿namespace BootCamp.Chapter.Items
+{
+    public class Headpiece
+    {
+        private string _name;
```

Comment:
Codacy found an issue: [Make '_name' 'readonly'.](https://app.codacy.com/manual/Almantask/CSharp-From-Zero-To-Hero/pullRequest?prid=5776099)